### DISCUSSION TITLE
my chia app 1.1.5 on Mac Pro not work,  keep on the screen "connecting to wallet"

### DISCUSSION THREAD

**Discussion by rupan-ji** [Chia-Network/chia-blockchain#5092] [Support]
chia 1.1.5 can work on windows, MacOS Catalina 10.15.7, but not successful on macOS Sierra 10.12.6. need support from professional

**Comment by lvcivs** [comment]
It was the same for me when I tried it on an old iMac running High Sierra. Unfortunately it looks like older versions of MacOS are not properly supported.

**Reply by arunzn** [reply]
+1

**Comment by arunzn** [comment]
+1

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
